apps/extension/src/ui/apps/dashboard/routes/Networks/useNetworkType.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from "react"
+import { useSearchParams } from "react-router-dom"
+
+export const useNetworkType = () => {
+  const [searchParams, setSearchParams] = useSearchParams()
+  const type = searchParams.get("type")
+  const defaultNetworkType = type === "polkadot" ? "polkadot" : "ethereum"
+
+  const [networkType, setNetworkType] = useState<"polkadot" | "ethereum">(defaultNetworkType)
+  useEffect(() => {
```

Comment:
Not a big fan of this tbh, is there a need to write to the search params as well as just read them?